calm/draft/2024-08/meta/control.json

```diff
@@ -0,0 +1,68 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://raw.githubusercontent.com/finos/architecture-as-code/main/calm/draft/2024-08/meta/control.json",
+  "title": "Common Architecture Language Model Controls",
+  "defs": {
+    "control-requirement": {
+      "type": "object",
+      "properties": {
+        "control-id": {
+          "type": "string"
+        },
+        "name": {
+          "type": "string"
+        },
+        "description": {
+          "type": "string"
+        }
+      },
+      "required": [
+        "control-id",
+        "name",
+        "description"
+      ]
+    },
+    "evidence": {
+      "type": "object",
+      "properties": {
```

Comment:
I think we need to add a control-configuration-uri or control-requirement-uri in the evidence object so we know what the evidence is for?